doc/user/pimv6.rst

```diff
@@ -373,6 +385,18 @@ General multicast routing state
    Display total number of S,G mroutes and number of S,G mroutes
    installed into the kernel for all vrfs.
 
+.. clicmd:: show ipv6 pim bsr
+
+   Display current bsr, its uptime and last received bsm age.
+
+.. clicmd:: show ipv6 pim bsrp-info
+
+   Display group-to-rp mappings received from E-BSR.
+
+.. clicmd:: show ipv6 pim bsm-database
+
+   Display all fragments ofstored bootstrap message in user readable format.
```

Comment:
Please correct, ofstored here. I think you mean stored here.

Reply:
Addressed